src/backend/executor/nodeMaterial.c

```diff
@@ -41,14 +41,16 @@
 #include "postgres.h"
```

Comment:
I've been thinking that we should use a more meaningful name for all "tmp" or "temp"?